utils/bazel/llvm-project-overlay/third-party/BUILD.bazel

```diff
@@ -19,36 +14,20 @@ config_setting(
     flag_values = {":llvm_enable_zstd": "true"},
 )
 
-cc_library(
+cc_library_wrapper(
     name = "zstd",
-    srcs = select({
-        ":llvm_zstd_enabled": glob([
-            "lib/common/*.c",
-            "lib/common/*.h",
-            "lib/compress/*.c",
-            "lib/compress/*.h",
-            "lib/decompress/*.c",
-            "lib/decompress/*.h",
-            "lib/decompress/*.S",
-            "lib/dictBuilder/*.c",
-            "lib/dictBuilder/*.h",
-        ]),
-        "//conditions:default": [],
-    }),
-    hdrs = select({
+    defines = select({
         ":llvm_zstd_enabled": [
-            "lib/zdict.h",
-            "lib/zstd.h",
-            "lib/zstd_errors.h",
+            "LLVM_ENABLE_ZSTD=1",
+            "ZSTD_MULTITHREAD",
         ],
         "//conditions:default": [],
     }),
-    defines = select({
+    visibility = ["//visibility:public"],
```

Comment:
nit: instead of per-target visibility, can we keep this public at the package level? i.e. `package(default_visibility = ["//visibility:public"])`?